tree/tree/inc/TTree.h

```diff
@@ -529,6 +533,7 @@ class TTree : public TNamed, public TAttLine, public TAttFill, public TAttMarker
    virtual void            SetDirectory(TDirectory* dir);
    virtual Long64_t        SetEntries(Long64_t n = -1);
    virtual void            SetEstimate(Long64_t nentries = 1000000);
+           ROOT::TIOFeatures SetIOFeatures(const ROOT::TIOFeatures &);
```

Comment:
May add a using TIOFeatures = ROOT::TIOFeatures; above to short and align this line.